app/scripts/views/confirm_account_unlock.js

```diff
@@ -137,9 +137,8 @@ function (Cocktail, FormView, BaseView, Template, p, AuthErrors, Constants,
           email,
           password,
           self.relier,
-          {
-            reason: self.fxaClient.SIGNIN_REASON.ACCOUNT_UNLOCK
-          }
+          /* HACK: See eslint/eslint#1801 */
```

Comment:
You can remove the comment— it's not really a hack but an acceptable way to format an object literal with one property.